cogs5e/sheetManager.py

```diff
@@ -385,6 +385,66 @@ async def character(self, ctx, *, name: str = None):
 
         await ctx.send(f"Active character changed to {char.name}.", delete_after=20)
 
+    @character.command(name='server')
+    async def server_character(self, ctx, *, name: str = None):
+        """Switches the active character on the current server."""
+        if(ctx.guild is not None):
+            user_characters = await self.bot.mdb.characters.find({"owner": str(ctx.author.id)}).to_list(None)
+            if not user_characters:
+                return await ctx.send('You have no characters.')
+            
+            if name is None:
+                active_character: Character = await Character.from_ctx(ctx)
+                return await ctx.send(f'Currently active: {active_character.name} ({", ".join(active_character.active_context(ctx))}, `{active_character.sheet_type}`)')
+            
+            selected_char = await search_and_select(ctx, user_characters, name, lambda e: e['name'],
+                                                    selectkey=lambda e: f"{e['name']} (`{e['upstream']}`)")
+            
+            char = Character.from_dict(selected_char)
```

Comment:
If this command becomes a toggle for the global active character, all of this can be removed.